presto-cassandra/src/main/java/com/facebook/presto/cassandra/CassandraClientModule.java

```diff
@@ -97,8 +96,7 @@ public static CassandraSession createCassandraSession(
         requireNonNull(config, "config is null");
         requireNonNull(extraColumnMetadataCodec, "extraColumnMetadataCodec is null");
 
-        Cluster.Builder clusterBuilder = Cluster.builder()
-                .withProtocolVersion(ProtocolVersion.V3);
+        Cluster.Builder clusterBuilder = Cluster.builder();
```

Comment:
Looks like there is a PR to make the protocol version configurable: https://github.com/prestodb/presto/pull/7955